Tests are not passing

I usually do pull requests to keep the packaged version of mautrix-whatsapp in Alpine Linux repos updated.

Related:
https://gitlab.alpinelinux.org/alpine/aports/-/tree/master/testing/mautrix-whatsapp

However the test step is failing:

```
[eskuero@4cx4 mautrix-whatsapp]$ go test ./...
# maunium.net/go/mautrix-whatsapp/database
database/portal.go:87:19: fmt.Sprintf format %@ has unknown verb @
?       maunium.net/go/mautrix-whatsapp [no test files]
?       maunium.net/go/mautrix-whatsapp/config  [no test files]
```

Seems like it's related to 40ea8a8324eb07075ff4154388f14adbd1d56306 when fmt was introduced. If I remove the tests from the build process the bridge seems to work correctly but ideally we would be able to keep them.

--- a/database/portal.go
+++ b/database/portal.go
@@ -84,7 +84,7 @@ func (pq *PortalQuery) GetAllByJID(jid types.JID) []*Portal {
 }
 
 func (pq *PortalQuery) FindPrivateChats(receiver types.JID) []*Portal {
-	return pq.getAll(fmt.Sprintf("SELECT %s FROM portal WHERE receiver=$1 AND jid LIKE '%@s.whatsapp.net'", portalColumns), receiver.ToNonAD())
+	return pq.getAll(fmt.Sprintf("SELECT %s FROM portal WHERE receiver=$1 AND jid LIKE '%%@s.whatsapp.net'", portalColumns), receiver.ToNonAD())
 }
 
 func (pq *PortalQuery) FindPrivateChatsNotInSpace(receiver types.JID) (keys []PortalKey) {

